data-raw/prepare_data.R

```diff
@@ -0,0 +1,36 @@
+library(austraits)
+library(arrow)
+library(dplyr)
+source("R/a-helper.R")
+
+# Prepare the data for the portal
+
+# Lite version
+austraits:::austraits_5.0.0_lite |> 
+  prepare_data_for_portal("inst/extdata/austraits/austraits-5.0.0-lite")
+
+# Full version
+austraits_6.0.0 <- 
+  austraits::load_austraits(version = "6.0.0", path = "inst/extdata/austraits", update = FALSE)
+
+austraits_6.0.0 |> 
+  prepare_data_for_portal("inst/extdata/austraits/austraits-6.0.0-full")
+
+# Mid version
+
+# Create an intermediate sized version, to reduce size of the database
+# Only has a subset of taxa and only core traits
+## Only core traits
+traits <- arrow::read_csv_arrow("inst/extdata/austraits/trait_groups_for_portal.csv") |> filter(!is.na(core_trait)) |> pull(trait)
```

Comment:
Nice work @dfalster!  Could you please add the namespaces here for `filter` and `pull` for clarity and easier debugging? Thanks!